api/transactions.js

```diff
@@ -607,7 +570,7 @@ function getAccountTransactions(api, options, callback) {
 
   function sortTransactions(transactions, async_callback) {
     transactions.sort(function(first, second) {
-      return compareTransactions(first, second, options.earliestFirst);
+      return utils.compareTransactions(first, second, !options.earliestFirst);
```

Comment:
Could do that, or just do it manually:
`function swapArguments(f) { return function(a, b) { return f(b, a); } }`
or just multiply the result of the compare function by `1` or `-1`